htmlSanityCheck-core/build.gradle

```diff
@@ -13,12 +13,11 @@ dependencies {
     testImplementation libs.slf4j.nop
     // jsoup is our awesome html parser, see jsoup.org
     implementation libs.jsoup
-
+    implementation group: 'net.ricecode', name: 'string-similarity', version: '1.0.0'
```

Comment:
Please add the version to `gradle/libs.versions.toml`

Reply:
done